rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -512,6 +512,50 @@ def test_get_summary_list_returns_total(self):
     self.assertEqual(response2['total'], response['total'])
     self.assertEqual(response2['total'], num_participants)
 
+  def test_get_summary_list_returns_offset_results(self):
+    num_participants = 10
+    SqlTestBase.setup_codes([PMI_SKIP_CODE], code_type=CodeType.ANSWER)
+    questionnaire_id = self.create_demographics_questionnaire()
+
+    # generate participants to count
+    for _ in range(num_participants):
+      # Set up participant, questionnaire, and consent
+      participant = self.send_post('Participant', {"providerLink": [self.provider_link]})
+      participant_id = participant['participantId']
+      with FakeClock(TIME_1):
+        self.send_consent(participant_id)
+      # Populate some answers to the questionnaire
+      answers = {
+        'race': RACE_WHITE_CODE,
+        'genderIdentity': PMI_SKIP_CODE,
+        'firstName': self.fake.first_name(),
+        'middleName': self.fake.first_name(),
+        'lastName': self.fake.last_name(),
+        'zipCode': '78751',
+        'state': PMI_SKIP_CODE,
+        'streetAddress': '1234 Main Street',
+        'city': 'Austin',
+        'sex': PMI_SKIP_CODE,
+        'sexualOrientation': PMI_SKIP_CODE,
+        'phoneNumber': '512-555-5555',
+        'recontactMethod': PMI_SKIP_CODE,
+        'language': PMI_SKIP_CODE,
+        'education': PMI_SKIP_CODE,
+        'income': PMI_SKIP_CODE,
+        'dateOfBirth': datetime.date(1978, 10, 9),
+        'CABoRSignature': 'signature.pdf',
+      }
+      self.post_demographics_questionnaire(participant_id, questionnaire_id, **answers)
+
+    response = self.send_get('ParticipantSummary?_count=10')
+    # Pass offset query parameter
+    response2 = self.send_get('ParticipantSummary?_count=5&_offset=2')
```

Comment:
Please add tests that show:

1) What happens if you request more participants than remain (e.g. offset = 6, count = 5)
2) What happens if you set an offset bigger than the number of participants (e.g. offset = 10)

Reply:
Sure, will add these test cases.